src/styles/blocks.scss/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  width: 166px;
+  max-height: 16px;
+  margin: 0;
+  padding-top: 17px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../../images/star.svg);
+    background-repeat: no-repeat;
+    background-size: 80%;
+    align-self: flex-end;
+  }
+  &__star.active {
+    background-image: url(../../images/star-active.svg);
+  }
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__reviews {
+    margin: 0;
+    padding-left: 18px;
```

Comment:
still not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/84e56a91-54a4-42f3-86df-8e58d03d889b)